infra.bs

```diff
@@ -820,11 +819,11 @@ and monospace font.
 
 <p class=example id=example-string-notation>"<code>Hello, world!</code>" is a string.
 
-<p class=note>This is different from how the Unicode Standard defines "code unit". In particular it
-refers exclusively to how the Unicode Standard defines it for Unicode 16-bit strings. [[UNICODE]]
+<p class=note>This is different from how <cite>Unicode</cite> defines "code unit". In particular it
+refers exclusively to how <cite>Unicode</cite> defines it for Unicode 16-bit strings. [[UNICODE]]
 
 <p>A <a>string</a> can also be interpreted as containing <a>code points</a>, per the conversion
-defined in <a>The String Type</a> section of the JavaScript specification. [[!ECMA-262]]
+defined in <a>The String Type</a> section of <cite>JavaScript</cite>. [[!ECMA-262]]
```

Comment:
Again this is not the actual title, so I prefer leaving it as "the JavaScript specification"